punx/h5tree.py

```diff
@@ -22,6 +22,10 @@
 import logging
 import os
 import h5py
+try:  # hdf5plugin loads codecs used by h5py, don't need to call it below
+    import hdf5plugin
+except ImportError:
+    pass  # avoids unused-import report from flake8
```

Comment:
Why don't you move this to `punx/__init__.py`? Then you wouldn't need to repeat it in each module? I count this being the third time.

Reply:
Good idea

Reply:
The try..except is an attempt to quash the unused-import report from Codacy.  It does not accomplish that goal.

Reply:
Also removed the editing of `sys.path` from `punx/__init__.py`.  Was not needed.